haiku-apps/copynametoclipboard/copynametoclipboard-1.0.1.recipe

```diff
@@ -9,7 +9,10 @@ COPYRIGHT="2000 Werner Freytag
 	2012 Humdinger"
 LICENSE="MIT"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+srcGitRev="1da1d52429db53f15c3debf817e982d9ce6346b4"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1da1d52429db53f15c3debf817e982d9ce6346b4"
```

Comment:
What do I input to get the sha256sum? Is it the name of the recipe, contents of the recipes, etc?